server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/CloudDLMJobStore.java

```diff
@@ -55,26 +135,89 @@ public void bindGeneratorToJob(String s, TaskGenerator taskGenerator) throws SQL
 
     @Override
     public JobStatistic getJobStatistic(String s) throws JobException {
-        return null;
+        return new JobStatistic();
     }
 
     @Override
     public void storeJobStatistic(JobMeta jobMeta) throws JobSqlException {
-
+        jobMeta.getJobStat().buildReportData();
     }
 
     @Override
     public List<TaskMeta> getTaskMeta(JobMeta jobMeta) throws SQLException {
+        if (enableBreakpointRecovery) {
```

Comment:
may bee add a  BreakpointRecoveryJobStore for handle enableBreakpointRecovery

Reply:
This is not necessary as this configuration is dynamic